ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -57,70 +56,73 @@
       #set ($view = 'past')
     #end
   #end
-#end
 
-#set($tabs = [])
-#macro(addAllDocsTab $tab)
-  #if($xwiki.hasAccessLevel('view', "$!xcontext.user", $tab.get('document')))
-    #set($discard = $tabs.add($tab))
-  #end
-#end
+  #set($tabs = [])
 
-#addAllDocsTab({'tabName' : 'past', 'idSuffix' : 'pastMeetings', 'translationKey' : 'meeting.pastMeetings.title'})
-#addAllDocsTab({'tabName' : 'future', 'idSuffix' : 'futureMeetings', 'translationKey' : 'meeting.futureMeetings.title'})
-#addAllDocsTab({'tabName' : 'calendar', 'idSuffix' : 'meetingsCalendar', 'translationKey' : 'contrib.meeting.calendarView.title'})
+  #set($discard = $tabs.add({'tabName' : 'past', 'idSuffix' : 'pastMeetings', 'translationKey' : 'meeting.pastMeetings.title'}))
+  #set($discard = $tabs.add({'tabName' : 'future', 'idSuffix' : 'futureMeetings', 'translationKey' : 'meeting.futureMeetings.title'}))
+  #set($discard = $tabs.add({'tabName' : 'calendar', 'idSuffix' : 'meetingsCalendar', 'translationKey' : 'contrib.meeting.calendarView.title'}))
 
-{{html}}
-&lt;div class="floatcontainer"&gt;
-  &lt;ul class="xwikitabbar"&gt;
-#foreach ($tab in $tabs)
-  &lt;li id="xwiki$tab['idSuffix']"#if($view == $tab['tabName']) class="active"#end&gt;&lt;a href="$doc.getURL('view', "view=$tab['tabName']&amp;amp;$!param")"&gt;$services.localization.render($tab['translationKey'])&lt;/a&gt;&lt;/li&gt;
-#end
-  &lt;/ul&gt;
-&lt;/div&gt;
-{{/html}}
-
-#if ($view == 'calendar')
-  {{html wiki="true"}}
-    {{include reference='Meeting.CalendarView'/}}
-  {{/html}}
-#else
-  #set($docextras = [])
-  #set ($discard = $xwiki.ssfx.use('uicomponents/widgets/userpicker/userPicker.css'))
-  #set ($discard = $xwiki.ssfx.use('uicomponents/pagination/pagination.css', 'true'))
-  #set ($columnsProperties = {
-    'doc.title': {'type': 'text', 'size': 10, 'link': 'view'},
-    'leader': {'type': 'text', 'size': 10, 'html': true},
-    'place': {'type': 'text', 'size': 10},
-    'startDate': {'type': 'text', 'size': 10},
-    'endDate': {'type': 'text', 'size': 10},
-    'durationDisplay': {'type': 'text', 'filterable': true, 'sortable': false, 'html': true},
-    'status': {'type': 'list', 'size': 10},
-    '_actions': {'sortable': false, 'filterable': false, 'html': true, 'actions': ['delete', 'edit']}
-  })
-  #set ($options = {
-    'className': 'Meeting.Code.MeetingClass',
-    'translationPrefix': 'meeting.livetable.',
-    'tagCloud': true,
-    'rowCount': 15
-  })
-  #set ($currentDate = $xwiki.jodatime.dateTime.millis)
+  {{html clean="false" wiki="true"}}
+  &lt;div&gt;
+    &lt;ul class="nav nav-tabs" role="tablist"&gt;
+    #foreach ($tab in $tabs)
+      &lt;li #if($view == $tab['tabName']) class="active"#end
+        ## When clicking on the Calendar tab, the calendar itself is not displayed properly. Resizing the window will
```

Comment:
Maybe because the Calendar is initialized inside a hidden tab (e.g. it relies on some computed widths that are 0), but I don't understand why we didn't have this problem when using the tab HTML structure from the page index (and we have it now with Bootstrap).

Reply:
The Page Index html structure that I used before would reload the page and include only the Calendar view, thus it was initialized normally. Right now, using Bootstrap, I include/initialize everything on the page at the same time. If Calendar was the default tab when accessing Meetings.WebHome, then it would be displayed properly.